src/libraries/System.Runtime.Experimental/ref/System.Runtime.Experimental.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We need to pack this project.  I think we can use the new CSProj packing if we add this to https://github.com/dotnet/runtime/blob/22c1e12edcf861b64ceff0a7d8f3ab309b7cd8dc/src/libraries/libraries-packages.proj#L7
And follow some of the steps here: https://github.com/dotnet/runtime/issues/53202
And use `<BuildOutputTargetFolder>ref</BuildOutputTargetFolder>` to make a ref-only package.

Reply:
Its very non-obvious to me what is required here. I don't see any steps detailing how to build a ref only package.

It looks like, from some of the linked PRs I may need to manually include: `<PackageReference Include="NuGet.Build.Tasks.Pack" Version="$(NuGetBuildTasksPackVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />` and set `IsPackable=true`, but it isn't clear if there is anything else that's required

Reply:
Well, you can take a look at the branch I put together where I hacked this up. I ended up using a small mini-project which redirects the build task to trigger the pack task on the particular binary, then you could reference it from thhe libraries-packages project or do what I did which was reference it directly from the subsets file. Also, the detail about `BuildOutputTargetFolder` is the critical one for creating a ref only package.

Reply:
You aren't going to find any obvious tutorial for a ref package as you're the first one doing that.  I listed the things I think you need to do.  Here's my attempt at it, might be enough, the package looked correct but I didn't do a full build: https://github.com/ericstj/runtime/commit/044d532eee1d0f5ec765a7c95dfcfaf3c7f1d89b